src/api/services/apiKeyService.ts

```diff
@@ -0,0 +1,41 @@
+import { Participant } from '../entities/Participant';
+import { CreatedApiKeyDTO } from './adminServiceHelpers';
+
+export type ApiKeySecretsDTO = {
+  plaintextKey: string;
+  secret: string;
+  name: string;
+};
+
+export const createdApiKeyToApiKeySecrets = (apiKey: CreatedApiKeyDTO): ApiKeySecretsDTO => {
+  // eslint-disable-next-line camelcase
+  return {
+    plaintextKey: apiKey.plaintext_key,
+    secret: apiKey.authorizable.secret,
+    name: apiKey.authorizable.name,
+  };
+};
+
+export const getApiRoles = async (participant: Participant) => {
+  const participantCurr = await Participant.query()
```

Comment:
perhaps rename to `participantWithApiRoles`? the `Curr` is probably unnecessary here, thouhgh I'm also okay with `currentParticipantWithApiRoles`

Reply:
Renamed it too `participantWithApiRoles`